.github/workflows/ubuntu.yml

```diff
@@ -113,12 +120,13 @@ jobs:
           - elixir-version: 'master'
             otp-version: '23.1'
             os:  'ubuntu-20.04'
-          - elixir-version: 'master'
+          - elixir-version: 'main'
             otp-version: '25'
             os:  'ubuntu-20.04'
             version-type: 'strict'
           - gleam-version: 'nightly'
             otp-version: '24'
+            os: 'ubuntu-latest'
```

Comment:
I believe this should've been broken, but can't understand how we didn't see it before. In any case, on the README, the Ubuntu build is broken and there were recently issues with running actions on GitHub actions [like so](https://github.com/erlef/setup-beam/actions/runs/3066558491), which was hiding bugs.